sdk/servicebus/service-bus/package.json

```diff
@@ -93,6 +92,7 @@
   },
   "devDependencies": {
     "@azure/eslint-plugin-azure-sdk": "^2.0.1",
+    "@azure/ms-rest-nodeauth": "^0.9.2",
```

Comment:
Now that aadutils.ts file is deleted, we don't need `@azure/ms-rest-nodeauth` in tests either right?

Reply:
agree, updated